code/server/routes/api.js

```diff
@@ -2,21 +2,13 @@ var _express = require('express');
 
 var router = _express.Router();
 var app = _express();
-var User = require('../models/User.js');
-var UserMatches = require('../models/UserMatches.js');
+var UserDAO = require('../data_access/UserDataAccess.js');
+var UserMatchDAO = require('../data_access/UserMatchDataAccess.js');
 var authenticator = require("../mixins/authenticator.js");
 
-var injectUser = function(user) {
-	if (user) {
-		User = user;
-	}
-};
-
-var injectLikes = function(likes) {
-	if (likes) {
-		UserMatches = likes;
-	}
-};
+// Initialize models to use DB
+UserDAO.init('db');
+UserMatchDAO.init('db');
```

Comment:
I wish I could think of a nicer way to do it, but I feel like we need to have some sort of inject function so that we can initialize the stubs. This is like the one time I've ever missed java spring dependency injection in my entire life lol